internal/brokers/requests/requests.go

```diff
@@ -0,0 +1,9 @@
+// Package requests contains the possible requests for the authentication.
+package requests
+
+const (
+	// WaitSet is the response when the authentication needs waiting.
+	WaitSet = "true"
+	// SkipSet is the response when the authentication is skipped.
+	SkipSet = "true"
```

Comment:
These changes have nothing to do with the PR subject. You either move them to a separate one and then we'll discuss them there or we drop it entirely.

Reply:
I understand if you referred to changing the behavior, but not cosmetic things.

However, let me revert this too.